extensions/wikia/Wall/WallMessage.class.php

```diff
@@ -448,7 +448,7 @@ public function setNotifyEveryone( $notifyEveryone ) {
 					wfMessage( 'wall-message-update-highlight-summary' )->inContentLanguage()->text(),
 					false, true );
 				$rev = $this->getArticleComment()->mLastRevision;
```

Comment:
I'm eliminating the city ID argument from places where we shouldn't need it.  Were we need it are times when the Revision data is on a wiki that is not the current wiki.  Since we already have a revision object created here in ```$rev``` we have what we need and the city ID is not necessary.